internal/cli/cli.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/innosat-mats/level1a-platform/internal/filewriter"
```

Comment:
@joakimmoller @local-minimum 
får inte det här med importen av egen modul att funka på github. lokalt funkar det.
Ser ni vad det är jag missat. 

får detta fel när vi kör go.yml ska köras på github.
Run go get -v -t -d ./...
github.com/innosat-mats/level1a-platform (download)
created GOPATH=/home/runner/go; see 'go help gopath'
package github.com/innosat-mats/level1a-platform/internal/filewriter: cannot find package "github.com/innosat-mats/level1a-platform/internal/filewriter" in any of:
	/opt/hostedtoolcache/go/1.13.10/x64/src/github.com/innosat-mats/level1a-platform/internal/filewriter (from $GOROOT)
	/home/runner/go/src/github.com/innosat-mats/level1a-platform/internal/filewriter (from $GOPATH)
get "gonum.org/v1/hdf5": found meta tag get.metaImport{Prefix:"gonum.org/v1/hdf5", VCS:"git", RepoRoot:"https://github.com/gonum/hdf5"} at //gonum.org/v1/hdf5?go-get=1
gonum.org/v1/hdf5 (download)
##[error]Process completed with exit code 1.

Reply:
du måste ha din lokala kopia av repot på exakt rätt plats på disk dvs `~/go/github.com/innosat-mats/level1a-platform`

Reply:
nu funkar det. efetr att jag installerat lokalt ock lagt till go.mod  go.sum.